openfisca_core/legislations.py

```diff
@@ -30,9 +30,14 @@ def __init__(self, name, instant, variable_name = None):
         self.name = name
         self.instant = instant
         self.variable_name = variable_name
-        message = u'Legislation parameter "{}" was not found at instant "{}"'.format(name, instant)
+        message = u'\n\nThe parameter "{}"'.format(name)
         if variable_name is not None:
-            message += u' by variable "{}"'.format(variable_name)
+            message += u' requested by variable "{}" '.format(variable_name)
+        message += (
+            u" was not found in the {2} tax and benefit system.\n"
+            u"If this code used to work and suddenly does not,"
+            u"it is most probably linked to a discrepancy in the Country Package code.\n"
```

Comment:
It's worth mentionning that the sentence "If this code used to work and suddenly does not..." was introduced because variables were often renamed, breaking users code suddenly.
It does _not_ necessarily aims at become the general template of all our error messages 😉 .